features/about/build.gradle.kts

```diff
@@ -8,7 +8,7 @@ plugins {
 android {
     compileSdkVersion(AndroidSDK.compile)
 
-    defaultConfig {
+    this.defaultConfig {
```

Comment:
@michaelbukachi will this be the new way to define this?

Reply:
@wangerekaharun oops. I was just testing something. It can be changed.

Reply:
Cool no worries